debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleOffsetContext.java

```diff
@@ -71,7 +71,10 @@ public OracleOffsetContext(OracleConnectorConfig connectorConfig, Scn scn, Strin
         sourceInfo.setLcrPosition(lcrPosition);
         sourceInfoSchema = sourceInfo.schema();
 
-        this.snapshotScn = snapshotScn;
+        // Snapshot SCN is a new field and may be null in cases where the offsets are being read from
+        // and older version of Debezium.  In this case, we need to explicitly enforce Scn#NULL usage
```

Comment:
Grr I really dislike the fact that our checkstyle complains about double spaces after the end of a sentence.  It's so common place for me to write this way and it always gets me everytime...